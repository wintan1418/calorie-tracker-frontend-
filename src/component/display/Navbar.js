import { useHistory } from 'react-router-dom';
import PropTypes from 'prop-types';

const Navbar = ({ title }) => {
  const history = useHistory();

  const logOut = () => {
    sessionStorage.removeItem('token');
    history.push('/log_in');
  };