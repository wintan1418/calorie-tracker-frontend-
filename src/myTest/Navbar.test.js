 import React from 'react';
 import renderer from 'react-test-renderer';
 import Navbar from '../component/display/Navbar';

 it('renders accurately', () => {
   const tree = renderer.create(<Navbar title="more"/>).toJSON();
   expect(tree).toMatchSnapshot();

 });