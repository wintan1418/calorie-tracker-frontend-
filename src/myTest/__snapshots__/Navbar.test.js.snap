// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders accurately 1`] = `
<nav>
  <div
    className="n-title"
  >
    more
  </div>
  <button
    onClick={[Function]}
    type="button"
  >
    <i
      className="fas fa-power-off"
    />
  </button>
</nav>
`;
